src/components/IOUConfirmationList.js

```diff
@@ -0,0 +1,248 @@
+
+    // Holds data related to IOU view state, rather than the underlying IOU data.
+    iou: PropTypes.shape({
+
+        // Whether or not the IOU step is loading (creating the IOU Report)
+        loading: PropTypes.bool,
+    }),
+};
+
+const defaultProps = {
+    iou: {},
+    onUpdateComment: null,
+    comment: '',
+};
+
+class IOUConfirmationList extends Component {
+    /**
+     * Returns the sections needed for the OptionsSelector
+     *
+     * @param {Boolean} maxParticipantsReached
+     * @returns {Array}
+     */
+    getSections() {
+        const sections = [];
+
+        if (this.props.hasMultipleParticipants) {
+            const formattedMyPersonalDetails = getIOUConfirmationOptionsFromMyPersonalDetail(
+                this.props.myPersonalDetails,
+
+                // convert from cent to bigger form
+                `$${this.calculateAmount(true) / 100}`,
+            );
+            const formattedParticipants = getIOUConfirmationOptionsFromParticipants(this.props.participants,
+                `$${this.calculateAmount() / 100}`);
+
+            sections.push({
+                title: 'WHO PAID?',
+                data: formattedMyPersonalDetails,
+                shouldShow: true,
+                indexOffset: 0,
+            });
+            sections.push({
+                title: 'WHO WAS THERE?',
+                data: formattedParticipants,
+                shouldShow: true,
+                indexOffset: 0,
+            });
+        } else {
+        // $ should be replaced by currency symbol once available
+            const formattedParticipants = getIOUConfirmationOptionsFromParticipants(this.props.participants,
+                `$${this.props.iouAmount}`);
+
+            sections.push({
+                title: 'TO',
+                data: formattedParticipants,
+                shouldShow: true,
+                indexOffset: 0,
+            });
+        }
+        return sections;
+    }
+
+    /**
+     * Gets splits for the transaction
+     *
+     * @returns {Array}
+     */
+    getSplits() {
+        const splits = this.props.participants.map(participant => ({
+            email: participant.login,
+
+            // we should send in cents to API
+            amount: this.calculateAmount(),
+        }));
+
+        splits.push({
+            email: this.props.myPersonalDetails.login,
+
+            // the user is default and we should send in cents to API
+            amount: this.calculateAmount(true),
+        });
+        return splits;
+    }
+
+    /**
+     * Gets participants list for a report
+     *
+     * @returns {Array}
+     */
+    getParticipants() {
+        const participants = this.props.participants.map(participant => participant.login);
+        participants.push(this.props.myPersonalDetails.login);
+        return participants;
+    }
+
+    /**
+     * Returns selected options with all participant logins
+     * @returns {Array}
+     */
+    getAllOptionsAsSelected() {
```

Comment:
I'm a bit confused by this. 

>  Returns selected options with all participant logins

What exactly does 'selectedOptions' mean?

Reply:
I get options. Selected meaning the IOU participants?

Reply:
I meant that it selects all IOUParticipants for the display in IOUConfirm’s split flow. I will revise the comment to explain it better :)